docs/DeveloperGuide.md

```diff
@@ -591,42 +589,116 @@ testers are expected to do more *exploratory* testing.
 
 ### Launch and shutdown
 
-1. Initial launch
+1. Initial launch.
 
-   1. Download the jar file and copy into an empty folder
+    1. Download the jar file and copy it into an empty folder.
 
-   1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
+    2. Double-click the jar file. <br>
+       Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
 
-1. Saving window preferences
+2. Saving window preferences.
 
-   1. Resize the window to an optimum size. Move the window to a different location. Close the window.
+    1. Resize the window to an optimum size. Move the window to a different location. Close the window.
 
-   1. Re-launch the app by double-clicking the jar file.<br>
+    2. Re-launch the app by double-clicking the jar file.<br>
        Expected: The most recent window size and location is retained.
 
-1. _{ more test cases …​ }_
+### Adding a batchmate
+
+1. Adding a batchmate to the data.
+
+    1. Test case: `add n/John t/john`.<br>
+       Expected: A new batchmate with the name John and telegram handle john is added at the bottom of the students pane in MassLinkers.
+       The status message indicates that John has been successfully added.
+
+    2. Prerequisite: A batchmate with the name Tom is already in MassLinkers. <br>
+       Test case: `add n/Tom t/tom`.<br>
+       Expected: No batchmate is added. The status message indicates that the batchmate already exists in MassLinkers.
+
+    3. Test case: `add n/John t/john i/AI g/john`.<br>
+       Expected: A new batchmate with the name John, telegram handle john, interests AI and GitHub john is added at the bottom of the students pane in MassLinkers.
+       The status message indicates that John has been successfully added.
+
+    4. Test case: `add n/John`.<br>
+       Expected: No batchmate is added. The status message indicates that the command is of an invalid format.
+
+    5. Other incorrect `add` commands to try: `add`, `add n/John t/john x/invalid`, `...` (where x is any invalid prefix).<br>
+       Expected: Similar to previous.
+
+
+### Deleting a batchmate
+
+1. Deleting a batchmate while all batchmates are being shown.
+
+    1. Prerequisites: List all batchmates using the `list` command. Multiple batchmates in the list.
+
+    2. Test case: `delete 1`.<br>
+       Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message.
```

Comment:
Replace contact with batchmate here? 🤔 
Add "is" here, i.e. "Details of the deleted contact __is__ shown in the status message".